2021-05-21 00:13:41,638 ERROR      chat_server_v2 : Error - [Errno 48] Address already in use
2021-05-21 00:17:25,475 ERROR      chat_client_v2 : [Errno 61] Connection refused
2021-05-21 00:28:35,066 ERROR      chat_server_v2 : Error - [Errno 48] Address already in use
2021-05-21 00:33:44,138 ERROR      chat_server_v2 : Error - [Errno 48] Address already in use
2021-05-21 00:34:59,103 ERROR      chat_server_v2 : Error - [Errno 48] Address already in use
2021-05-22 00:47:40,326 DEBUG      log_config : 2021.47.22 00:47 функция create_server вызвана из функции <module>
2021-05-22 00:47:40,327 INFO       chat_server_v2 : Create server at "":8885
2021-05-22 00:49:32,967 DEBUG      log_config : 2021.49.22 00:49 функция create_server вызвана из функции <module>
2021-05-22 00:49:32,969 INFO       chat_server_v2 : Create server at "":8885
2021-05-22 00:49:41,488 INFO       chat_server_v2 : Принят запрос от ('127.0.0.1', 58743)
2021-05-22 00:49:41,488 DEBUG      log_config : 2021.49.22 00:49 функция handle_data вызвана из функции receive_data
2021-05-22 00:49:41,489 INFO       chat_server_v2 : Сообщение от клиента: {"action": "presence", "time": "2021-05-22 00:49:41", "type": "status", "user": {"account_name": "oleg", "status": "Enable"}}
2021-05-22 00:52:00,418 ERROR      chat_server_v2 : Error - [Errno 48] Address already in use
2021-05-22 00:52:30,857 INFO       chat_server_v2 : Create server at "":8886
2021-05-22 00:57:34,150 INFO       chat_server_v2 : Create server at "":8886
2021-05-22 00:58:11,662 INFO       chat_server_v2 : Create server at "":8886
2021-05-22 01:00:08,246 INFO       chat_server_v2 : Create server at "":8886
2021-05-22 01:01:15,248 DEBUG      log_config : 2021.01.22 01:01 функция create_server вызвана из функции <module>
2021-05-22 01:01:15,249 INFO       chat_server_v2 : Create server at "":8886
